src/sw.js

```diff
@@ -0,0 +1,24 @@
+self.addEventListener('push', function(event) {
```

Comment:
I think this files will be overwritten by the `sw-precache` plugin we're using. Need to dig into it a little more.